src/UserMenu/UserMenu.tsx

```diff
@@ -95,7 +91,11 @@ export default function UserMenu({
  *
  * Provides custom styling and converts username to max 2 initials
  */
-const UserAvatar = ({ username, ...rest }: UserMenuProps) => {
+interface UserAvatarProps {
```

Comment:
typescript props should live in UserMenu.types.ts even for internal sub components.